### PR TITLE
rename 'vm_map_max_count' into 'vm_max_map_count'

### PR DESCRIPTION
**Pull request by wlalele**

### CODE REVIEW

**Comment by jasontedor** [comment]
@wlalele Can you sign the CLA?

**Comment by wlalele** [comment]
@jasontedor I signed it 3 times and it is still red, can you relaunch the test manually ?

**Comment by jasontedor** [comment]
Thanks @wlalele. I ran the CLA checker manually and it's correct now.

File: docs/reference/setup/install/docker.asciidoc [hunk 1]
```diff
@@ -66,7 +66,7 @@ Depending on your platform:
 
 * Linux
 +
-The `vm_map_max_count` setting should be set permanently in /etc/sysctl.conf:
+The `vm_max_map_count` setting should be set permanently in /etc/sysctl.conf:
```
Comment (jasontedor) [inline docs/reference/setup/install/docker.asciidoc]: I think this should be changed to `vm.max_map_count` as well. Can you push that change?

**Review by jasontedor — CHANGES_REQUESTED** [review]
Thanks for the PR @wlalele. This is correct that the words are not in the correct order, but I think there's one more change to make while we are here.

**Comment by wlalele** [comment]
@jasontedor does it look like what you wanted ?

**Review by jasontedor — APPROVED** [review]
LGTM.

**Comment by jasontedor** [comment]
Thanks @wlalele!

**Pull request merged**